utils/chemiscope_input.py

```diff
@@ -210,3 +210,54 @@ def write_chemiscope_input(filename, frames, meta=None, extra=None, cutoff=None)
     else:
         with open(filename, 'w') as file:
             json.dump(data, file)
+
+helpstring = """
+    Command-line utility to generate an input for chemiscope  - an interactive 
+    structure-property explorer. Parses an input file containing atomic structures
+    using the ASE I/O module, and converts it into a JSON file that can be loaded in
+    chemiscope. Frame and environment properties must be written in the same file 
+    containing atomic structures: we recommend the extended xyz format, which is 
+    flexible and simple. In all cases, this utility will simply transfer to the
+    JSON input anything that is readable by ASE. 
+"""
+            
+if __name__ == '__main__':
+
+    # command-line execution. requires ASE IO module
+    import argparse
+    try:
+        import ase.io as ase_io
+    except:
+        raise ImportError("chemiscope_input needs ASE modules to parse structure inputs")
+    
+    parser = argparse.ArgumentParser(description=helpstring)
+    parser.add_argument('-i', '--input', type=str, required=True,
+                        help='Input file name.')
+    parser.add_argument('-o', '--output', type=str, 
+                        help='Chemiscope JSON output file.')
+    parser.add_argument('-c', '--cutoff', type=float, 
+                        help='Generate  atom-centred environments with the \
+                              given cutoff.')                        
+    parser.add_argument('--name', default="", type=str,
+                        help='Dataset name.')
+    parser.add_argument('--description', default="", type=str,
+                        help='Dataset description.')
+    parser.add_argument('--authors', nargs='*', type=str, default=[],
+                        help='List of dataset authors.')
+    parser.add_argument('--references', nargs='*', type=str, default=[],
+                        help='List of references for the dataset.')                 
+    args = parser.parse_args()
+
+    # reads filen with ASE    
+    frames = ase_io.read(args.input, ':')
+    
+    # determines output file name automatically if missing
+    output = args.output
+    if args.output is None:
+        output = args.input + "_chemiscope.json"
+    
+    write_chemiscope_input(output, frames, 
+        meta=dict(name=args.name, description=args.description,
+                authors=args.authors, references=args.references),
+        extra=None, cutoff=args.cutoff)
```

Comment:
Ideally, one should be able to flag automatic property imports. If sticking to an ase-like format, my thoughts would be parser arguments to flag atom- or structure-centered properties.

Reply:
A chemiscope file can contain both atom and structure properties, and if a file contains both, the viewer will default to atom properties. #1 is about allowing the user to display the other kind of properties.

Until #1 is implemented, I think having a flag to select atom or structure properites is a good solution. It can be implemented as 

```py
if args.only_structure:
    for frame in frames:
        frame.arrays = {}
elif args.only_atoms:
    for frame in frames:
        frame.info = {}
```

Reply:
I agree this would be nice to have as makeshift while we wait for #1 
My impression was actually that it only uses environment properties if one selects a cutoff value, am I off? 
Another thing that is annoying - but is more a problem with `write_chemiscope_input` than with the command-line wrapper - is the fact that it picks up the atomic numbers array as a property that should be stored in chemiscope.

Reply:
> My impression was actually that it only uses environment properties if one selects a cutoff value, am I off?

No, that's right, I remembered the logic wrongly (cf https://github.com/cosmo-epfl/chemiscope/blob/815eabec23c6e30910077d07d2cc5da54e6857b0/src/index.ts#L126), passing a `cutoff` value created the `environments` array.

> Another thing that is annoying - but is more a problem with write_chemiscope_input than with the command-line wrapper - is the fact that it picks up the atomic numbers array as a property that should be stored in chemiscope.

This is actually an ASE thing, which always create `frame.arrays["number"]`. We could ignore it in `write_chemiscope_input`, but then it is sometime useful, cf the CSD-1000 dataset. We could add a note on this subject in the documentation though.

Reply:
I would ignore arrays["numbers"]: if one wants that information, *e can add a string property with the element labels, or just duplicate that property calling it "atomic_number". more often than not I suspect this is not useful, and unfortunately seems you cannot remove "numbers" from arrays before you pass frames to chemiscope_input because it breaks the internal ASE handling of the frames.

Reply:
This convinced me =). I'll check if the code below still works then:

```py
if args.only_structure:
    for frame in frames:
        frame.arrays = {}
```